packages/web-components/@sfx/search-box/src/search-box.ts

```diff
@@ -184,3 +196,7 @@ export default class SearchBox extends Base {
     `;
   }
 }
+export interface SearchConfig {
```

Comment:
This interface isn't actually being used. Is the intention to use it as part of `emitSearchEvent()`?